frontend/src/modules/entitieslist/components/Entity.js

```diff
@@ -20,6 +20,8 @@ type Props = {
     selected: boolean,
     selectEntity: Function,
     getSiblingEntities: Function,
+    areFiltersApplied: Function,
+    parameters: Object,
```

Comment:
We should be as specific as possible when declaring type and avoid generic types like `Object`. Here, you should use `NavigationParams` type, which needs to be imported from `core/navigation`.

Reply:
I understand because the `NavigationParams` that will be imported will set the types to be what is exported out of the file `selector.js`

Reply:
and with a generic type like `Object`, it's not setting any rules for us to follow